### PR TITLE
added all examples for How I Leaped Forward My Jenkins Build with JFr…

### PR DESCRIPTION
**Pull request by talitz**
…og Pipelines blog post

### CODE REVIEW

**Comment by talitz** [comment]
I have read the CLA Document and I hereby sign the CLA

File: jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        stage ('Clone') {
+            steps {
+                git branch: 'master', url: "https://github.com/talitz/spring-petclinic-ci-cd-k8s-example.git"
+            }
+        }
+
+        stage ('Artifactory Configuration') {
+            steps {
+                rtServer (
+                    id: "talyi-artifactory",
```
Comment (eyalbe4) [inline jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/Jenkinsfile:14]: Let's use a more generic server ID, similarly to the other examples.

File: jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/ci-jenkins-cd-pipelines/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        }
+
+        stage ('Publish Build Info') {
+            steps {
+                rtPublishBuildInfo (
+                    serverId: "talyi-artifactory"
+                )
+            }
+        }
+
```
Comment (eyalbe4) [inline jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/ci-jenkins-cd-pipelines/Jenkinsfile:78]: Redundant new-line.

File: jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/ci-jenkins-cd-pipelines/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+        stage('Trigger JFrog Pipelines') {
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'incoming-web-hook-creds', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                   sh "curl -XPOST https://$USERNAME:$PASSWORD@talyi-pipelines-api.jfrog.io/v1/projectIntegrations/39/hook"
+                }
+            }
+        }
+
```
Comment (eyalbe4) [inline jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/ci-jenkins-cd-pipelines/Jenkinsfile:87]: Redundant new-line.

File: jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                    deployerId: "maven-deployer",
+                    resolverId: "maven-resolver"
+                )
+            }
+        }
+
+        stage ('Build Docker Image') {
+            steps {
+                script {
+                  sh "cd target && ls -la"
```
Comment (eyalbe4) [inline jenkins-examples/pipeline-examples/declarative-examples/jenkins-with-jfrog-pipelines/Jenkinsfile:52]: Let's remove this step.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Thanks for creating this PR!
See my inline comments. Notice that some of them also apply for other places in your code, and not just the specific lines.

**Comment by talitz** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Thanks @talitz!
There's something else I noticed that is missing. Can you please add your new example to the README in https://github.com/jfrog/project-examples/tree/master/jenkins-examples/pipeline-examples/declarative-examples?

**Comment by talitz** [comment]
@eyalbe4 - thanks. resolved!

**Comment by talitz** [comment]
Hi @eyalbe4 - anything else that should be added here?

**Review by eyalbe4 — APPROVED** [review]

**Comment by eyalbe4** [comment]
Thanks @talitz!
Approved and merged.
Please go ahead and create another PR, which updates your blog post link in the README.

**Pull request merged**